src/services/AssetService.ts

```diff
@@ -1,18 +1,25 @@
 // tslint:disable:max-classes-per-file
-import { SearchResult } from '../store';
 import { Endpoints, FlowEditorConfig, ContactProperties, AttributeType } from '../flowTypes';
 import axios, { AxiosResponse } from 'axios';
 import { FlowComponents } from '../store/helpers';
+import { dump } from '../utils';
+
+export interface Asset {
```

Comment:
just a head's up, `Simulator` has an `Asset` interface as well

Reply:
not for long it doesn't! simulator is going to need a bit of reworking for assetage as well, imagine it won't need it's own thing.